apps/models/app_types/custom.py

```diff
@@ -25,7 +25,7 @@ class CustomAppInstance(BaseAppInstance, SocialMixin, LogsEnabledMixin):
     )
 
     volume = models.ForeignKey(
-        "VolumeInstance", blank=True, null=True, related_name="%(class)s", on_delete=models.CASCADE
+        "VolumeInstance", blank=True, null=True, related_name="%(class)s", on_delete=models.SET_NULL
```

Comment:
Shouldn't this have generated migrations files?

Reply:
Yes indeed. Coming...